src/System/executable_query.rs

```diff
@@ -6,12 +6,14 @@ use crate::ModelObjects::component::Component;
 use crate::System::refine;
 use crate::System::save_component::combine_components;
 use crate::TransitionSystems::TransitionSystemPtr;
+use crate::ModelObjects::component::State;
 
 use super::extract_system_rep::SystemRecipe;
 use super::save_component::PruningStrategy;
 
 pub enum QueryResult {
     Refinement(bool),
+    Reachability(bool,Vec<String>), //Skal ændre String til Edge
```

Comment:
Maybe we should figure out the exact format of this before merging

Reply:
We are fixing this in another pull request. It is only for protobuf, however this change is for the cli parser which means this has nothing to do with this pull request.